src/components/controls3d/style/controls3d.less

```diff
@@ -7,7 +7,6 @@
   bottom: unit(@bt-bottom, px);
   margin-left: auto;
   margin-right: auto;
-  max-width: 250px;
```

Comment:
If you remove this, the div takes all the width of the window and you can't interact with  the map in this zone. can be annoying .